controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -224,7 +234,7 @@ module.exports = async (req, res) => {
         type: sequelize.QueryTypes.SELECT,
         replacements: {
           topics,
-          admin_user_id: process.env.BETTER_ADMIN_ID,
+          allUserIds,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

As mentioned above, ensure `allUserIds` is converted back to an array before being used in the SQL query.
```diff
- allUserIds,
+ allUserIds: Array.from(allUserIds),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->